bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -95,7 +95,8 @@ PreProcessor::PreProcessor(shared_ptr<MsgsCommunicator> &msgsCommunicator,
       maxReplyMsgSize_(myReplica.getReplicaConfig().maxReplyMessageSize - sizeof(ClientReplyMsgHeader)),
       idsOfPeerReplicas_(myReplica.getIdsOfPeerReplicas()),
       numOfReplicas_(myReplica.getReplicaConfig().numReplicas),
-      numOfClients_(myReplica.getReplicaConfig().numOfClientProxies),
+      numOfClients_(myReplica.getReplicaConfig().numOfClientProxies +
+                    myReplica.getReplicaConfig().numOfExternalClients),
```

Comment:
Preprocessor works only with external clients.

Reply:
so, remove numOfClientProxies? because before that he took only numOfClientProxies.

Reply:
Yes. Previously this number was expected to include all possible clients.

Reply:
For now, we keep using the numOfClientProxies parameter in order not to break the Preprocessor tests.